providers/rethinkdb.js

```diff
@@ -1,11 +1,11 @@
-const { Provider } = require("klasa");
+const { Provider, util: { mergeObjects } } = require("klasa");
 const rethink = require("rethinkdbdash");
 
 module.exports = class extends Provider {
 
     constructor(...args) {
         super(...args);
-        this.db = rethink(this.client.options.providers.rethinkdb || { db: "pengubot" });
+        this.db = rethink(this.client.options.providers.rethinkdb || { db: "pengubot", timeout: 100 });
```

Comment:
What does timeout do?

Reply:
Lmfao, `timeout` is in seconds. no wonder why you get errors with sharding

Reply:
Not really, when I had ot set to default aka 20 seconds it showed the error I sent yesterday.

Reply:
then go blame @kyranet

Reply:
What do I have to do with RethinkDB, @MrJacz? 😕

Reply:
@kyranet The timeout issue yukine was having with SG